bokehjs/src/coffee/core/util/bbox.coffee

```diff
@@ -1,13 +1,11 @@
-empty = () ->
-  return [[Infinity, -Infinity], [Infinity, -Infinity]]
-
 union = (a, b) ->
-  a[0][0] = Math.min(a[0][0], b[0][0])
-  a[0][1] = Math.max(a[0][1], b[0][1])
-  a[1][0] = Math.min(a[1][0], b[1][0])
-  a[1][1] = Math.max(a[1][1], b[1][1])
-  return a
+  r = {}
+  r.minX = Math.min(a.minX, b.minX)
+  r.maxX = Math.max(a.maxX, b.maxX)
+  r.minY = Math.min(a.minY, b.minY)
+  r.maxY = Math.max(a.maxY, b.maxY)
+  return r
 
 module.exports =
-  empty: empty
+  EMPTY: {minX: Infinity, minY: Infinity, maxX: -Infinity, maxY:-Infinity}
```

Comment:
This should be a function. Otherwise you will get this modified in-place sooner or later.